tools/ttmlir-opt/CMakeLists.txt

```diff
@@ -1,9 +1,17 @@
 get_property(dialect_libs GLOBAL PROPERTY MLIR_DIALECT_LIBS)
 get_property(conversion_libs GLOBAL PROPERTY MLIR_CONVERSION_LIBS)
-set(LIBS ${dialect_libs} ${conversion_libs} MLIROptLib MLIRTargetCpp TTMLIRStatic)
+set(LIBS ${dialect_libs} ${conversion_libs} StablehloBase TTNNTargetFlatbuffer MLIRTTIRDialect TTMLIRConversions MLIRTransformUtils MLIROptLib MLIRTargetCpp TTMLIRStatic CheckOps ChloCAPI ChloOps InterpreterOps StablehloAssemblyFormat StablehloBase StablehloBroadcastUtils StablehloCAPI StablehloLinalgTransforms StablehloOps StablehloPasses StablehloPortableApi StablehloRegister StablehloSerialization StablehloTOSATransforms StablehloTestUtils StablehloTypeInference Version VhloCAPI VhloOps VhloTypes)
+
+if (TTMLIR_ENABLE_STABLEHLO)
+	list(APPEND LIBS TTMLIRStableHLOToTTIR)
+endif()
+
 add_llvm_executable(ttmlir-opt ttmlir-opt.cpp)
 
+target_link_directories(ttmlir-opt PRIVATE ${TTMLIR_PROJECT_DIR}/build/lib)
+target_link_libraries(ttmlir-opt PRIVATE ${LIBS} libStablehloBase.a)
+target_link_directories(ttmlir-opt PRIVATE ${TTMLIR_TOOLCHAIN_DIR}/lib)
```

Comment:
We didn't need this in the past, it's redundant to the one specified at the top level cmake file.